CHANGES.rst

```diff
@@ -1,6 +1,29 @@
 Changes
 =======
 
+0.6.0 (YYYY-MM-DD)
+------------------
+
+* Fixed the ``extract_from`` spider parameter that wasn't working.
+
+* The *"www."* prefix is now removed when setting the spider's
+  :attr:`~scrapy.Spider.allowed_domains`.
+
+* The :attr:`zyte_common_items.ProductNavigation.nextPage` link won't be crawled
+  if :attr:`zyte_common_items.ProductNavigation.items` is empty.
+
+* :class:`zyte_common_items.Product` items that are dropped due to low probability
+  *(below 0.1)* are now logged in stats: ``drop_item/product/low_probability``.
+
+* :class:`zyte_spider_templates.pages.HeuristicsProductNavigationPage` now
+  inherits from :class:`zyte_common_items.AutoProductNavigationPage` instead of
+  :class:`zyte_common_items.BaseProductNavigationPage`.
+
+* Moved eCommerce specific code from :class:`zyte_spider_templates.spiders.base.BaseSpider`
+  to :class:`zyte_spider_templates.spiders.ecommerce.EcommerceSpider`.
```

Comment:
```suggestion
* Moved e-commerce code from :class:`zyte_spider_templates.spiders.base.BaseSpider`
  to :class:`zyte_spider_templates.spiders.ecommerce.EcommerceSpider`.
```

I think “e-commerce” is the right wording.

You could leave the “specific”, but then I think it would need to be hyphenated, yet “e-commerce-specific” although correct feels rather weird to me, specially since I don’t think it adds much here.